### PR TITLE
Adds tactical chairs

### PR DESCRIPTION
**Pull request by ghost**
:cl:
add: Adds tactical chairs
/:cl:

we removed tactical chairs years back, this PR makes chairs tactical again.

### CODE REVIEW

**Comment by notfrying1pans** [comment]
what exactly does it do

**Comment by SgtHunk** [comment]
> what exactly does it do

Make the game better.

**Comment by itseasytosee** [comment]
@notfrying1pans it sets the chairs layer to OBJ_LAYER and gives chairs a 1% chance put the prefix tactical infont of chairs you put down.

**Comment by Qustinnus** [comment]
i didnt change the layer just moved the var to the right place

**Review by Cyberboss — APPROVED** [review]

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 1]
```diff
@@ -25,6 +26,8 @@
 	. = ..()
 	if(!anchored)	//why would you put these on the shuttle?
 		addtimer(CALLBACK(src, .proc/RemoveFromLatejoin), 0)
+	if(prob(1))
```
Comment (Cyberboss) [inline code/game/objects/structures/beds_chairs/chair.dm:29]: image

Let's calm down a bit
```suggestion
	if(prob(0.2))
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by XDTM** [comment]
didn't prob() have trouble with values less than 1? or was it rand()?

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 2]
```diff
@@ -25,6 +26,8 @@
 	. = ..()
 	if(!anchored)	//why would you put these on the shuttle?
 		addtimer(CALLBACK(src, .proc/RemoveFromLatejoin), 0)
+	if(prob(0.2))
+		name = "tactical [name]"
```
Comment (Fikou) [inline code/game/objects/structures/beds_chairs/chair.dm:30]: make it layer above players noob

**Comment by Cyberboss** [comment]
> 
> 
> didn't prob() have trouble with values less than 1? or was it rand()?

I just tested prob, it works

File: code/game/objects/structures/beds_chairs/chair.dm (hunk 2 above)
Comment (Cyberboss) [reply code/game/objects/structures/beds_chairs/chair.dm:30]: no

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**